internal/support/notifications/sending_service_test.go

```diff
@@ -0,0 +1,70 @@
+/*******************************************************************************
+ * Copyright 2020 Technotects
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ *******************************************************************************/
+package notifications
+
+import (
+	"fmt"
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestBuildSmtpMessageNoContentType(t *testing.T) {
+	subject := uuid.New().String()
+	message := uuid.New().String()
+
+	result := buildSmtpMessage(subject, "", message)
+
+	require.NotNil(t, result)
+
+	stringResult := string(result)
+
+	assert.Equal(t, fmt.Sprintf("Subject: %s\r\n\r\n%s\r\n", subject, message), stringResult)
```

Comment:
Like to see `expected` set outside the assert for readability. Here and the other tests

Reply:
fixed both of these and rebased